### ISSUE TITLE
[embree3] build failure for triplet x64-windows-static

### ISSUE THREAD

**Issue by edsDev** [microsoft/vcpkg#8141]
**Host Environment**
- OS: Windows 10
- Compiler: MSVC 2019

**To Reproduce**
`./vcpkg install embree3`

**Failure logs** 
![image](https://user-images.githubusercontent.com/12378019/64747206-e51cbd80-d540-11e9-8fed-125087800003.png)
![image](https://user-images.githubusercontent.com/12378019/64747217-ee0d8f00-d540-11e9-802f-ddf60b494429.png)

**Additional context**
Sorry I don't know how to copy texts from Microsoft Terminal. I did manage to compile static library for embree3 manually, by the way.

**Comment by PhoebeHui** [comment]
@edsDev, thanks for posting this issue! I summit a fix for this.

**Comment by PhoebeHui** [comment]
@edsDev, the fix should work, I verified that, please see the failures message, and recommended fix.

**Comment by edsDev** [comment]
Your fix seems to simply remove the bin folder, which does eliminate the error message, but doesn't completely solve the problem here.

This port file toggles **EMBREE_STATIC_RUNTIME** ON on a static build
![image](https://user-images.githubusercontent.com/12378019/64931688-b36f6380-d86c-11e9-83a2-3ee69ba509f5.png)

But embree it fetches, v3.5.2 specifically, use **EMBREE_STATIC_LIB**
![image](https://user-images.githubusercontent.com/12378019/64931733-ff220d00-d86c-11e9-9dc0-20f98cbb335e.png)

Also, embree is a large library and would create quite a number of library files if you build it manually turning on **EMBREE_STATIC_LIB**. But in your branch (PhoebeHui:dev/Phoebe/fixembree3), vcpkg install command creates a single .lib file in size of a few kilobytes. Although I did't link my program to that file to test that, I don't expect it to work. I'm sorry I don't have time to try it right now as it would take an hour to build that on my computer.

I attempted to fix that myself, yet simply replacing **EMBREE_STATIC_RUNTIME** with **EMBREE_STATIC_LIB** does not solve the problem, too. It would bring further problems due to some non-conventional way embree handles its binary (and cmake file) output directory as I mentioned before.
